### PR TITLE
Merge List of statues 

### PR DESCRIPTION
**Pull request by Gina-Frankel**
- Changed names - list was replaced with menu
StatueListContainer >> StatueMenuContainer
StatueList >> StatueMenu
StatueListItem >>  StatueMenuItem

- Extracted StatueMenuItems from StatueMenu
- Tests for StatueMenuItems 
- Changed tests for StatueMenu so are checking to see if what StatueMenu renders is the equivalent of a StatueMenuItem component

### CODE REVIEW

File: src/components/StatueMenu.spec.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
```
Comment (hturnbull93) [inline src/components/StatueMenu.spec.js:2]: Import mount, but it isn't used?

File: src/components/StatueMenuContainer.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react";
+import { StatueMenu } from "./StatueMenu";
+
+export class StatueMenuContainer extends Component {
+  render() {
+    return (
+      <div>
+        <h1>Statues</h1>
```
[the remark this answers is not in the document]
Comment (Gina-Frankel) [reply src/components/StatueMenuContainer.js:8]: Changed it to a h2 for now ...

File: src/components/StatueMenu.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+import { StatueMenuItem } from "./StatueMenuItem";
```
[the remark this answers is not in the document]
Comment (Gina-Frankel) [reply src/components/StatueMenu.js:2]: Yes it needs be imported from index.js in the components directory. But index.js is not in this branch it is in another branch so once they are merged I will change the file location

File: src/components/StatueMenuItem.spec.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  const wrapper = shallow(<StatueMenuItem value={MockSlaverStatues} />);
+  expect(wrapper).toHaveLength(1);
+});
+
+it("returns the name of statue - Edward Colston", () => {
+  const MockSlaverStatues = {
+    name: "Edward Colston",
+  };
+  const wrapper = shallow(
+    <StatueMenuItem key={MockSlaverStatues.name} value={MockSlaverStatues} />
```
[the remark this answers is not in the document]
Comment (Gina-Frankel) [reply src/components/StatueMenuItem.spec.js:21]: Oh ok... thanks!

File: src/components/StatueMenu.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (hturnbull93) [reply src/components/StatueMenu.js:2]: Gotcha

**Pull request merged**